app/controllers/v1/sessions_controller.rb

```diff
@@ -0,0 +1,13 @@
+class V1::SessionsController < ApplicationController
+  def create
+    user = User.where(email: params[:email]).first
+
+    if user&.valid_password?(params[:password])
+      render json: user.as_json(only: %i[id email], status: :created)
```

Comment:
At the moment, we are only rendering the user with email. Let's return the jwt authentication token so the user can add it to their requests.